sql/timeseries/swControlledPages.sql

```diff
@@ -1,23 +1,18 @@
 #standardSQL
 SELECT
-  SUBSTR(_TABLE_SUFFIX, 0, 10) AS date,
-  UNIX_DATE(CAST(REPLACE(SUBSTR(_TABLE_SUFFIX, 0, 10), '_', '-') AS DATE)) * 1000 * 60 * 60 * 24 AS timestamp,
-  IF(ENDS_WITH(_TABLE_SUFFIX, 'desktop'), 'desktop', 'mobile') AS client,
-  ROUND(
-    SUM(
-      IF(IFNULL(
-        JSON_EXTRACT(payload, '$._blinkFeatureFirstUsed.Features.ServiceWorkerControlledPage'),
-        JSON_EXTRACT(payload, '$._blinkFeatureFirstUsed.Features.990')
-      ) IS NOT NULL, 1, 0)
-    ) * 100 / COUNT(0),
-    2
-  ) AS percent
+  REGEXP_REPLACE(yyyymmdd, r'(\d{4})(\d{2})(\d{2})', '\\1_\\2_\\3') AS date,
+  UNIX_DATE(CAST(REGEXP_REPLACE(yyyymmdd, r'(\d{4})(\d{2})(\d{2})', '\\1-\\2-\\3') AS DATE)) * 1000 * 60 * 60 * 24 AS timestamp,
+  client,
+  SUM(IF(id = '990' OR feature = 'ServiceWorkerControlledPage', num_urls, 0)) AS num_urls,
+  ROUND(SUM(IF(id = '990' OR feature = 'ServiceWorkerControlledPage', num_urls, 0)) / total_urls * 100, 5) AS percent
 FROM
-  `httparchive.pages.*`
+  `httparchive.blink_features.usage`
```

Comment:
Is this table updated by the time the reports are generated? Or is it expected to lag by a month?

Reply:
Like all the other blink features queries this one will be delayed. But not by a month, as we rerun the reports on the 2nd of the month specifically because this data is available then.

Eventually we'll rewrite these all to use the `all` tables at which point this will no longer be an issue but until then I don't see why this SW query needs to be treated differently.

Reply:
Just asking what the expected behavior is. Sounds like the lag is WAI, and you're right that it's less than a month.